### DISCUSSION TITLE
Ukrainian language for mobile app [Auth]

### DISCUSSION THREAD

**Discussion by Damglador** [ente-io/ente#3136] [Enhancements]
It's translated on Crowdin, I guess it's for desktop version. Would be cool to have Ukrainian language also for mobile app, if you need translators, I can help, I've worked on translation for Dome Keeper, Deadlink and working on translation for Roboquest as a volunteer.

**Comment by dnred** [comment]
Contact them on Crowdin.

**Reply by Damglador** [reply]
I think they've already added it since then, so there's no more need to (just checked my Ente Auth)
